docs/community/README.md

```diff
@@ -0,0 +1,132 @@
+# How to Get Involved
```

Comment:
nit: the sidebar shows "Welcome to the Knative community" versus "How to Get Involved", should we update the title here?

Reply:
Fixed this in `nav.yml`.